src/renderer.ts

```diff
@@ -346,7 +367,7 @@ const richLinkStyles = css`
 
 const RichLink = (props: { url: string; linkText: string }): ReactElement =>
     styledH('aside', { css: richLinkStyles },
-        styledH('a', { href: props.url }, [h('h1', null, props.linkText), 'Read more'])
+        styledH('a', { href: props.url }, [h('h1', null, props.linkText), h('p', null, 'Read more')])
```

Comment:
I know this is wrapped in a link, but should we make this a `<button>` semantically?